WebRoot/js/zimbraMail/calendar/controller/ZmCalViewController.js

```diff
@@ -1548,7 +1581,7 @@ function(date, duration, roll) {
 			this.show(ZmId.VIEW_CAL_WEEK);
 		}
 		if (ZmId.VIEW_CAL_MONTH == this._currentViewType) {
-			title = this._viewMgr.getCurrentView().getShortCalTitle();
+			title = this._viewMgr.getCurrentView().getCalTitle();
```

Comment:
i think we can get rid of this if condition
as we are already getting title from current view on line 1541 (currentView.getCalTitle())